src/Hl7.Fhir.ElementModel/TypedElementOnSourceNode.cs

```diff
@@ -18,6 +18,9 @@ namespace Hl7.Fhir.ElementModel
 {
     internal class TypedElementOnSourceNode : ITypedElement, IAnnotated, IExceptionSource, IShortPathGenerator
     {
+        private readonly string xHtmlInstancetype = "xhtml";
```

Comment:
we normally use the style `private const string XHTML_INSTANCETYPE = "xHtml"`

Reply:
Of course. I changed it.